### ISSUE TITLE
FocusTrapZoneProps should be exported from the main package?

### ISSUE THREAD

**Issue by sushruth** [microsoft/fluent-ui-react#1725]
`FocusTrapZoneProps` can currently be imported like so - 

```ts
import { FocusTrapZoneProps } from '@stardust-ui/react/dist/es/lib/accessibility/FocusZone';
```

Instead would be more appropriate to import like so - 

```ts
import { FocusTrapZoneProps } from '@stardust-ui/react'
```

**Comment by sophieH29** [comment]
Done https://github.com/stardust-ui/react/pull/1795
